tests/ui_tests/app/page/common/base_page.py

```diff
@@ -163,7 +167,7 @@ def assert_displayed_elements(self, locators: list) -> None:
             assert self.is_element_displayed(loc), f"Locator {loc.name} isn't displayed on page"
 
     def check_element_should_be_hidden(
-        self, locator: Locator, timeout: Optional[int] = None
+        self, locator: Union[Locator, WebElement], timeout: Optional[int] = None
```

Comment:
Hereinafter. It will be better if we have the possibility to pass one of separated arguments
```suggestion
        self, locator: Optional[Locator] = None, web_element: Optional[WebElement] = None, timeout: Optional[int] = None
```

Or I don't understand use cases?

Reply:
but one of them is required, they are not optional

Reply:
We can do a check, that one of the required attr passed. Or leave one args for two types (as it now) and rename it, because locator != WebElement

Reply:
changed to element